apps/website/src/pages/about/tech/commands.tsx

```diff
@@ -220,13 +225,28 @@ const AboutTechPage: NextPage = () => {
                           <pre>
                             <code className="text-sm">
                               {signature(command)}
+                              <CopyButton
+                                onClick={() => {
+                                  copyToClipboard(signature(command));
+                                  setSelectedCommand(command.name);
+                                  setTimeout(() => {
+                                    setSelectedCommand("");
+                                  }, 2000);
```

Comment:
The timeouts *should* be cleared, so we'd probably need a reference for the running timeout somewhere. Not sure if its worth. Its just a bit scuffed if you click a bunch of copy buttons.

Reply:
If we don't want to fix this properly now, can we leave a FIXME in the code for it?

Reply:
I think a better way to do this actually would be to store a Map of command name -> timeout, so that if you click a bunch in quick succession they each timeout on their own, and if you click the same one again we can clear the old timeout for it. So, not a shared state/timeout.

Reply:
Did it by moving the logic into a hook/component. So each is tracking its own state.